### PR TITLE
update readme and contributing

### PR DESCRIPTION
**Pull request by noot**
- update readme to say we use wasmer (not life)
- update broken spec link in contributing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Contribution Guidelines
+1# Contribution Guidelines
```
Comment (ansermino) [inline CONTRIBUTING.md]: Why `1`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 Thanks for checking out our Polkadot Runtime Implementation! We're excited to hear and learn from you.
 
 We've put together the following guidelines to help you figure out where you can best be helpful. The Web3 foundation has a comprehensive collection of [Polkadot Resources](https://github.com/w3f/web3/blob/537a2518c24e96b05ceadd9f31348669e72b8841/docs/layer_1/platforms/polkadot.md) for both part-time and core contributors to the project in order to get up to speed.
 
-Additionally, the [Polkadot Specification Doc](https://github.com/w3f/polkadot-spec/blob/master/spec.md) serves as the primary specification, however it is currently in its final draft status so things may be subject to change.
+Additionally, the [Polkadot Runtime Specification](https://research.web3.foundation/en/latest/web/viewer.html?file=../pdf/polkadot_re_spec.pdf) serves as the primary specification, however it is currently in its final draft status so things may be subject to change.
```
Comment (ansermino) [inline CONTRIBUTING.md]: While we're in here -- I think this should just say `draft status`, its far from final :sweat_smile:

File: CONTRIBUTING.md (hunk 1 above)
Comment (noot) [reply CONTRIBUTING.md]: typo woops

File: CONTRIBUTING.md (hunk 2 above)
Comment (noot) [reply CONTRIBUTING.md]: will change

**Pull request merged**